src/edgeProxy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "edgeproxy",
+    "version": "0.0.0",
+    "private": true,
+    "scripts": {
+        "buildLib": "rm -rf public/lib; mkdir public/lib && cp -rf node_modules/infusion public/lib/infusion",
+        "start": "npm run buildLib; node edgeProxy.js"
```

Comment:
This script still contains a non-portable ; operator

Reply:
Good catch, @amb26. It's now addressed.